po/zh_CN.po

```diff
@@ -0,0 +1,552 @@
+
+#, c-format
+msgid "%s used"
+msgstr "已使用 %s"
+
+#, c-format
+msgid ""
+"Last minute: %0.02f\n"
+"Last 5 minutes: %0.02f\n"
+"Last 15 minutes: %0.02f\n"
+"Processes/threads: %d active out of %d."
+msgstr ""
+"最近一分钟： %0.02f\n"
+"最近五分钟： %0.02f\n"
+"最近一刻钟： %0.02f\n"
+"进程/线程： %d ，活跃 %d。"
+
+#, c-format
+msgid ""
+"Monitored partitions: %s\n"
+"\n"
+"Read: %s\n"
+"Write: %s"
+msgstr ""
+"监控分区： %s\n"
+"\n"
+"读取： %s\n"
+"写入： %s"
+
+#, c-format
+msgid ""
+"Current: %.1f °C\n"
+"Critical: %.1f °C"
+msgstr ""
+"当前温度： %.1f °C\n"
+"临界温度： %.1f °C"
+
+#, c-format
+msgid "Current: %.1f °C"
+msgstr "当前温度： %.1f °C"
+
+#, c-format
+msgid "Command line is empty."
+msgstr "命令行为空。"
+
+#, c-format
+msgid "Unable to execute command."
+msgstr "无法执行命令。"
+
+#, c-format
+msgid "Command has exited with status code %d."
+msgstr "命令已退出，退出码为 %d。"
+
+#, c-format
+msgid "Command did not return valid numbers."
+msgstr "命令没有返回有效的数字。"
+
+#, c-format
+msgid ""
+"Command: %s\n"
+"ERROR: %s"
+msgstr ""
+"命令： %s\n"
+"错误： %s"
+
+#, c-format
+msgid ""
+"Command: %s\n"
+"Results: (%.3lf, %.3lf, %.3lf, %.3lf)"
+msgstr ""
+"命令： %s\n"
+"结果： (%.3lf, %.3lf, %.3lf, %.3lf)"
+
+#, c-format
+msgid "ERROR: %s"
+msgstr "错误： %s"
+
+#, c-format
+msgid "Unable to execute the following command line: '%s'"
+msgstr "无法执行以下的命令行： '%s'"
+
+msgid "Processor"
+msgstr "处理器"
+
+msgid "Memory"
+msgstr "内存"
+
+msgid "Network"
+msgstr "网络"
+
+msgid "Swap"
+msgstr "SWAP"
+
+msgid "Load average"
+msgstr "平均负载"
+
+msgid "Disk"
+msgstr "磁盘"
+
+msgid "Temperature"
+msgstr "温度"
+
+msgid "Parametric"
+msgstr "参数"
+
+#, c-format
+msgid "%d pixel"
+msgstr "%d 像素"
+
+#, c-format
+msgid "%d milliseconds"
+msgstr "%d 毫秒"
+
+msgid "Import color scheme"
+msgstr "导入配色方案"
+
+msgid "_Cancel"
+msgstr "取消（&C）"
+
+msgid "_Open"
+msgstr "打开（&O）"
+
+msgid "Color scheme format is incorrect. Unable to import."
+msgstr "配色方案格式错误，无法导入。"
+
+msgid ""
+"Color scheme was created by an incompatible version of Multiload-ng. Unable "
+"to import."
+msgstr "配色方案由不兼容的 Multiload-ng 版本建立。无法导入。"
+
+msgid "Export color scheme"
+msgstr "导出配色方案"
+
+msgid "_Save"
+msgstr "保存（&S）"
+
+msgid "Error exporting color scheme."
+msgstr "导出配色方案出错。"
+
+#, c-format
+msgid "Command line is valid. Retrieved %d numbers."
+msgstr "命令行无效，Retrieved %d numbers."
+
+msgid "(Custom)"
+msgstr "（自定义）"
+
+#. xgettext: this is a rate format (eg. 2 MiB/s)
+#, c-format
+msgid "%s/s"
+msgstr "%s/s"
+
+#. xgettext: single-letter form for "days" - e.g. 3 days -> 3d
+msgid "d"
+msgstr "天"
+
+#. xgettext: single-letter form for "hours" - e.g. 3 hours -> 3h
+msgid "h"
+msgstr "小时"
+
+#. xgettext: single-letter form for "minutes" - e.g. 3 minutes -> 3m
+msgid "m"
+msgstr "分"
+
+#. xgettext: single-letter form for "seconds" - e.g. 3 seconds -> 3s
+msgid "s"
+msgstr "秒"
+
+#, c-format
+msgid "Unable to open the following url: '%s'"
+msgstr "无法打开以下网址：‘%s’"
+
+#. xgettext: Translator Full Name <translator_email@example.com>
+msgid "translator-credits"
+msgstr "translator-credits"
```

Comment:
There should be your name and email ( `Dee.H.Y <dongfengweixiao@hotmail.com>, 2016` ) in place of "translator-credits"